src/sentry/services/hybrid_cloud/user/service.py

```diff
@@ -81,6 +82,20 @@ def get_by_username(
         """
         pass
 
+    @rpc_method
+    @abstractmethod
+    def get_organizations(
+        self,
+        *,
+        user_id: int,
+        only_visible: bool = False,
+    ) -> List[RpcOrganizationSummary]:
+        """Get summary data for all organizations of which the user is a member.
+
+        The organizations may span multiple regions.
+        """
+        pass
```

Comment:
This could just as easily go in `OrganizationMappingService`, but IMHO `UserService` is the second-most intuitive place to go looking for this (first being `OrganizationService`).

From `user_service.get_organizations`, it seems like it should be clear enough that you're getting "the user's organizations", i.e., the ones of which the user is a member. I considered naming it "get_organizations_with_membership" or something, but that invites confusion about whether you're getting back the member object or the org object. (The implementation actually queries for both. It wouldn't be too hard to return a list of `Tuple[RpcOrganizationSummary, RpcOrganizationMemberMapping]` if that would be useful.)

Reply:
👍